CONTRIBUTING.md

```diff
@@ -95,11 +106,18 @@ Array.should have_method(:new)
   e.message.should include("oops")
   e.cause.should == nil
 }
+```
 
-# To avoid! Instead, use an expectation testing what the code in the lambda does.
-# If an exception is raised, it will fail the example anyway.
+**To avoid!** Instead, use an expectation testing what the code in the lambda does.
+If an exception is raised, it will fail the example anyway.
```

Comment:
Probably I should deprecate `should_not raise_error`.
However, there are 176 occurrences in ruby/spec to fix first :/

Reply:
https://github.com/ruby/spec/issues/754